.github/workflows/test.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Install GraphicsMagick
```

Comment:
Added due to this error - https://github.com/ministryofjustice/peoplefinder/actions/runs/11363235492/job/31606739572